requirements/edx/base.txt

```diff
@@ -227,6 +227,7 @@ django==4.2.16
     #   enmerkar
     #   enmerkar-underscore
     #   event-tracking
+    #   forum
```

Comment:
I think you forgot to commit the changes in base.in. This might be because of a conflict resolution.

Reply:
Actually there are no changes in base.in, Forum requirement is set in github.in